### ISSUE TITLE
Favicons not passing real-favicon check

### ISSUE THREAD

**Issue by scriptPilot** [scriptPilot/app-framework#333]
![bildschirmfoto 2017-03-15 um 00 48 28](https://cloud.githubusercontent.com/assets/19615586/23927391/b333437a-0919-11e7-845b-6f15c3f96a51.png)

**Comment by scriptPilot** [comment]
Corrected, but still some errors. Contacted realfavicon team.

![bildschirmfoto 2017-03-18 um 17 35 48](https://cloud.githubusercontent.com/assets/19615586/24074009/5e2d45ca-0c01-11e7-8ec7-10eec01e989d.png)

**Comment by scriptPilot** [comment]
Response from realfavicon team:

Interesting issue. Internally RFG is using icotools to inspect favicon.ico. With https://app-framework.scriptpilot.de/favicon.ico , this is what I get :

philippe@ubuntu-14:~/remove_this$ icotool -l favicon.ico
favicon.ico: incorrect total size of bitmap (1064 specified; 1128 real)
--icon --index=1 --width=16 --height=16 --bit-depth=32 --palette-size=0
favicon.ico: skipping 4072 bytes of garbage at 1182
favicon.ico: incorrect total size of bitmap (9256 specified; 9640 real)
favicon.ico: premature end

philippe@ubuntu-14:~/remove_this$ identify favicon.ico
favicon.ico[0] ICO 16x16 16x16+0+0 32-bit sRGB 14.5KB 0.000u 0:00.000
favicon.ico[1] ICO 32x32 32x32+0+0 32-bit sRGB 14.5KB 0.000u 0:00.000
favicon.ico[2] ICO 48x48 48x48+0+0 32-bit sRGB 14.5KB 0.000u 0:00.000

In other words, icotools fails but ImageMagick succeeds. I don't what's wrong. This is the first time I'm reported this error. Unfortunately I won't be able to investigate this right now.

Well I guess you can ignore this issue... Sorry for the non-full-green report :-/
